tests/test_selection/test_shuffle_features.py

```diff
@@ -41,11 +41,12 @@ def test_sel_with_default_parameters(df_test):
     pd.testing.assert_frame_equal(sel.transform(X), Xtransformed)
 
 
-def test_regression_cv_3_and_r2(load_diabetes_dataset):
+def test_regression_cv_3_and_r2(df_test, load_diabetes_dataset):
```

Comment:
I don't think we are using df_test here. Are you still working on the tests and I got ahead of my game? If so, apologies.

Reply:
That's true. This is a left over from a check. I'll remove it.